ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -499,13 +495,14 @@
   </Hds::Form::Fieldset>
 
   {{#unless @model.isNew}}
-    <form.input
+    <Hds::Form::TextInput::Field
       @value={{@model.callback_url}}
-      @label={{t 'form.callback_url.label'}}
-      @helperText={{t 'form.callback_url.help'}}
-      readonly={{true}}
-      @disabled={{true}}
-    />
+      disabled={{true}}
```

Comment:
can we add `isoptional` attr here

Reply:
@DhariniJeeva `@isOptional` added back in last commit.  Thank you for the catch